dags/vz_populate_cr3_metadata.py

```diff
@@ -48,9 +48,9 @@
 @dag(
     dag_id="vz-populate-cr3-metadata",
     description="A DAG which populates CR3 metadata in the VZDB",
-    schedule_interval="*/5 8-10 * * *"
-    if DEPLOYMENT_ENVIRONMENT == "production"
-    else None,
+    schedule_interval=(
```

Comment:
So it tries to process everything at 8am, and if it completes it under 50 min, all green squares, then it tries again to pick up new records (?) at 9a, 10a, 11a?

Reply:
That is correct -- essentially, it's a non-op every time it runs except for the very first time it runs after CR3s have been downloaded by the dev doing that. I have gone with this try-every-hour thing because it's not known when the dev is going to do the downloading.

And if it happens to get fired up during the very short window of CR3 downloads, that's fine, it'll get the tail of the group at the top of the next hour.

Reply:
ah! so if someone doesn't run the cr3 downloads before 11a CT then they would have to go in and trigger this dag themselves

Reply:
would there be any downside of running this every hour from 8a-5p central? just so we don't have to remember to run it manually

Reply:
No, not really, it's a nothing of a non-op to do that. I did the noon thing purely to stay aligned to the way it used to be where it was morning only. I can shift it to hourly during the whole business day.

Us devs won't let this encourage us to do cr3 downloads later in the day 😉

Reply:
😉